blocks/Button/test.js

```diff
@@ -0,0 +1,224 @@
+/* eslint-env mocha */
+
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import sinon from 'sinon';
+
+import Button from './';
+
+describe('Button', () => {
+    it('is a button', () => {
+        const button = shallow(<Button>button</Button>);
+
+        expect(button.is('button.button')).to.be.true;
+    });
+
+    it('accepts name prop', () => {
+        const button = shallow(<Button name="my-button">button</Button>);
+
+        expect(button.prop('name')).to.equal('my-button');
+    });
+
+    it('accepts type prop', () => {
+        const button = shallow(<Button type="submit">button</Button>);
+
+        expect(button.hasClass('button_type_submit')).to.be.true;
+    });
+
+    it('accepts view prop', () => {
+        const button = shallow(<Button view="action">button</Button>);
+
+        expect(button.hasClass('button_view_action')).to.be.true;
+    });
+
+    it('accepts className prop', () => {
+        const button = shallow(<Button className="my-button">button</Button>);
+
+        expect(button.hasClass('my-button')).to.be.true;
+    });
+
+    it('accepts checked prop', () => {
+        const button = shallow(<Button checked>button</Button>);
+
+        expect(button.hasClass('button_checked')).to.be.true;
+
+        button.setProps({ checked: false });
+
+        expect(button.hasClass('button_checked')).to.be.false;
+    });
+
+    it('accepts disabled prop', () => {
+        const button = shallow(<Button disabled>button</Button>);
+
+        expect(button.prop('disabled')).to.be.true;
+        expect(button.hasClass('button_disabled')).to.be.true;
+
+        button.setProps({ disabled: false });
+
+        expect(button.prop('disabled')).to.be.falsy;
```

Comment:
JTBM(@narqo): chai doesn't have `to.be.falsy` [in the API](http://chaijs.com/api/bdd/#method_ok) :(